packages/wallets/src/components/FlowField/FlowTextField.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+import WalletTextField, { WalletTextFieldProps } from '../Base/WalletTextField/WalletTextField';
+import { Field, FieldProps } from 'formik';
+
+export interface TFlowFieldProps extends WalletTextFieldProps {
+    isInvalid?: WalletTextFieldProps['isInvalid'];
+    name: string;
+}
+
+const FlowTextField: FC<TFlowFieldProps> = ({ isInvalid = 'false', name, ...rest }) => {
+    return (
+        <Field name={name}>
+            {({ field, form, meta }: FieldProps) => {
+                return (
+                    <WalletTextField
+                        {...rest}
```

Comment:
as per comments in other PRs :) ...rest in props list always gives me headache :)
as I never know what props are actually being passed (so I cannot just read the code, and code is being read way more often than being written, so its important). Also, if I change parent component props, they will be passed down to this component unecessarily, potentially causing side effects / unexpected results.

Reply:
The `FlowTextField` component is just a wrapper to the `Field` Formik component and our input text field component `WalletTextField`, the `{...field}` is mandatory for `Field` component to track the changes in the input field and sync it to the `Formik` component in `FlowProvider`, whilst the `{...rest}` is just to allow the consumer to pass in `WalletTextField` component props, its impersonating the `WalletTextField` component with the additional functionality of Formik state syncing and handling for input fields

Reply:
@adrienne-deriv I understand that ...field required for Formik, thus only mentioned ...rest :)
simple scenario: I add prop to ComponentA, which then adds it to ComponentB without me knowing about that. 

Its bypassing typescript limitation and component interface and interface segregation rules, also potentially causing number of unecessary re-renders just because random property within parent componet have changed...